### PR TITLE
build script for network tests added, readme updated

### PR DESCRIPTION
**Pull request by gleb-urvanov**

### CODE REVIEW

File: scripts/run-test-chain.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+sed -i 's/"runtimeUpgradeProposalGracePeriod":.*/"runtimeUpgradeProposalGracePeriod": 0,/' ./chainspec.json
+sed -i 's/"setElectionParametersProposalGracePeriod":.*/"setElectionParametersProposalGracePeriod": 0,/' ./chainspec.json
+sed -i 's/"textProposalGracePeriod":.*/"textProposalGracePeriod": 0,/' chainspec.json
+sed -i 's/"setContentWorkingGroupMintCapacityProposalGracePeriod":.*/"setContentWorkingGroupMintCapacityProposalGracePeriod": 0,/' chainspec.json
+sed -i 's/"setLeadProposalGracePeriod":.*/"setLeadProposalGracePeriod": 0,/' chainspec.json
+sed -i 's/"spendingProposalGracePeriod":.*/"spendingProposalGracePeriod": 0,/' chainspec.json
+sed -i 's/"evictStorageProviderProposalGracePeriod":.*/"evictStorageProviderProposalGracePeriod": 0,/' chainspec.json
+sed -i 's/"setStorageRoleParametersProposalGracePeriod":.*/"setStorageRoleParametersProposalGracePeriod": 0,/' chainspec.json
+yes | cargo run --release -p joystream-node -- purge-chain --dev
+cargo run --release -p joystream-node -- --dev
```
Comment (mnaamani) [inline scripts/run-test-chain.sh]:
```suggestion
cargo run --release -p joystream-node -- --chain ./chainspec.json
```

To use the custom chainspec file that was produced.

**Review by mnaamani — CHANGES_REQUESTED** [review]
Nice trick, just needs small fix to use the edited chainspec.json file, maybe also add the chainspec.json file to .gitignore ?

File: scripts/run-test-chain.sh (hunk 1 above)
Comment (gleb-urvanov) [reply scripts/run-test-chain.sh]: Fixed, also removed build from the script

**Comment by mnaamani** [comment]
Included in https://github.com/Joystream/joystream/pull/399

**Pull request closed without merging**